hyperbook/book/spiele/mississippi queen/regeln.md

```diff
@@ -0,0 +1,225 @@
+
+![](Orangene_Linie.png)
+
+## 4 Der Zug
+
+Ein Zug besteht aus einer oder mehreren Aktionen. In einem Zug müssen
+(falls der Zug nicht auf einer Sandbank endet) durch die Aktionen
+insgesamt alle Bewegungspunkte (bestimmt durch die Geschwindigkeit
+des Schiffes) verbraucht werden. Die verschiedenen Aktionen sind:
+
+### 4.1 Beschleunigungsaktion
+
+Eine Beschleunigung kann nur als erste Aktion eines Zuges ausgeführt
+werden. Die Beschleunigung um eine Geschwindigkeitseinheit pro
+Zug ist frei, jede Beschleunigung um mehr als 1 kostet für jeden
+weiteren Geschwindigkeitspunkt eine Kohleeinheit. Möchte ein
+Spieler beispielsweise mit einer aktuellen Geschwindigkeit von 2 auf Geschwindigkeit 4 beschleunigen, kostet dies eine Kohleeinheit. Die
+maximale Geschwindigkeit ist 6, die niedrigste 1.
+Auf gleiche Weise kann auch abgebremst, also die Geschwindigkeit
+verringert werden. Dies wird der Einfachheit halber als
+Beschleunigungsaktion mit negativer Beschleunigung behandelt.
+
+### 4.2 Erzeugung der Bewegungspunkte
+
+Bevor nun eine der folgenden Aktionen durchgeführt werden kann,
+bekommt der Dampfer entsprechend seiner aktuellen Geschwindigkeit
+Bewegungspunkte. Diese Bewegungspunkte müssen durch die folgenden
+Aktionen aufgebraucht werden, am Ende des Zuges dürfen also keine
+Bewegungspunkte mehr vorhanden sein.
+
+### 4.3 Drehaktion
+
+Eine Drehaktion je Zug des Spielers um eine Einheit (also um 60◦
+) ist frei.
+Jede weitere Drehung erfordert eine Kohleeinheit (Ausnahme Abdrängen,
+siehe 4.4). Auf Sandbänken kann nicht gedreht werden.
+
+### 4.4 Abdrängaktion
+
+Endet eine Aktion auf einem Feld mit dem gegnerischen Dampfer,
+muss darauf eine Abdrängaktion folgen. Ein Spieler kann den Gegner
+auf ein beliebig angrenzendes, jedoch nicht direkt hinter dem Spieler
+liegendes, begehbares Feld abdrängen (Wasserfelder, Sändbänke und
+Baumstammfelder gelten als begehbar). Eine Abdrängaktion kostet einen
+Bewegungspunkt, zwei, falls auf ein Baumstammfeld abgedrängt wird.
+Ist die Geschwindigkeit des auf ein Baumstammfeld abgedrängten Bootes
+größer als 1, wird dessen Geschwindigkeit um 1 reduziert. Es darf nicht
+von einer Sandbank aus abgedrängt werden. Der abgedrängte Spieler
+bekommt eine zusätzliche freie Drehung für seinen nächsten Zug. Wurde
+der Spieler auf eine Sandbank abgedrängt, entfällt diese freie Drehung und
+die Geschwindigkeit des abgedrängten Spielers wird auf 1 reduziert.
+
+### 4.5 Bewegungsaktion
+
+Eine Bewegungsaktion erfolgt in die derzeitige Bewegungsrichtung des
+Schiffes. Sie kann nur über und auf passierbare (siehe 4.4) Felder erfolgen.
+Sie darf niemals durch ein vom Gegner besetztes Feld oder eine Sandbank
+gehen, darf jedoch auf dem Feld des Gegners oder einer Sandbank enden.
+Es ist weiterhin nicht erlaubt, durch das gesamte, zuletzt aufgedeckte
+Segment in ein noch nicht aufgedecktes Feld zu ziehen.
+
+### 4.6 Kombination von Aktionen
+
+Solange die Regeln der einzelnen Aktionsarten eingehalten werden, können
```

Comment:
`Der Aktionsablauf in einem Zug ist also: Beschleunigungsaktion -> Erzeugung der Bewegungspunkte -> Beliebige Kombination der restlichen Aktionen, solange die Aktionspunkte ausreichen und die Regeln eingehalten werden`
Das sollte die Reihenfolge noch klarer machen, kannst du aber auch anders umschreiben.